pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Purpose
```

Comment:
move this files to .github directory

check the documentation. In the future, there can be multiple PR repo for this repo. ( For news and  for the developments )

Reply:
Okay